ts/features/messages/screens/MessageRouterScreen.tsx

```diff
@@ -146,27 +168,55 @@ export const MessageRouterScreen = (
     shouldNavigateBackAfterPushNotificationInteraction
   ]);
 
+  useHeaderSecondLevel({
+    goBack: onCancelCallback,
+    supportRequest: true,
+    title: ""
+  });
+
+  if (isLoading) {
+    return (
+      <View
+        style={[
+          styles.loaderContainer,
+          { paddingBottom: safeAreaInsets.bottom }
+        ]}
+      >
+        <ContentWrapper>
+          <View style={styles.loaderSpinner}>
+            <LoadingSpinner size={48} />
+          </View>
+          <VSpacer size={16} />
+          <H4 style={styles.loaderText}>
+            {I18n.t("messageDetails.loadingText")}
+          </H4>
+        </ContentWrapper>
```

Comment:
why not use `LoadingScreenContent` here?

Reply:
Did not know about it, replaced!